src/foraging_gui/Foraging.py

```diff
@@ -2461,12 +2867,24 @@ def _Open(self,open_last = False,input_file = ''):
             fname=input_file
             self.fname=fname
         if fname:
+            # Load mat file
             if fname.endswith('.mat'):
                 Obj = loadmat(fname)
+
+            # Load the new_style json file
```

Comment:
lets not use "new_style" and "old style" references, since that will get confusing as time goes on

Reply:
many "checkpoint format" and "full session format" ?